src/main/java/codeu/util/MessageCleaner.java

```diff
@@ -0,0 +1,65 @@
+package codeu.util;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.regex.*;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document.OutputSettings;
+import org.jsoup.safety.Whitelist;
+
+public class MessageCleaner {
+
+	public static String clean(String messageContent){
```

Comment:
You could consider adding tests for this class? I don't feel too strongly though, so it's up to you!